BlueprintUI/Sources/Element/ElementContent+StrictSPL.swift

```diff
@@ -176,21 +191,51 @@ public struct StrictNeutralLayout: SingleChildLayout {
 }
 
 
-class StrictLayoutNode: StrictLayoutable {
+class StrictProposalCaptureNode: StrictLayoutable {
```

Comment:
It might be nice to add a quick doc comment here, but we can do that later if you're worried about rebasing the stack!